### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
Per https://github.com/OISF/suricata/pull/7708#issuecomment-1537214528

Make sure these boxes are signed before submitting your Pull Request -- thank you.

- [x] I have read the contributing guide lines at https://suricata.readthedocs.io/en/latest/devguide/codebase/contributing/contribution-process.html
- [x] I have signed the Open Information Security Foundation contribution agreement at https://suricata.io/about/contribution-agreement/
- [ ] I have updated the user guide (in doc/userguide/) to reflect the changes made (if applicable)

Link to [redmine](https://redmine.openinfosecfoundation.org/projects/suricata/issues) ticket:

Describe changes:

<details><summary>changes</summary>

* additional
* built into
* collision
* conjunction
* delimiter
* encountered
* envelope
* fine-grained
* forty
* gitlab
* identify
* inconsistent
* insensitive
* into
* invalid
* issuerdn
* length
* more
* negation
* output
* param
* position
* propagate
* received
* serve them if we already have something from server
* structure
* switch
* synchronise
* the
* thresh
* turn
* unusual
* workaround
</details>

Before https://github.com/jsoref/suricata/actions/runs/4908756330#summary-13296803535
After https://github.com/jsoref/suricata/actions/runs/4908756356#summary-13296803567 (yes, I didn't update my metadata a final time -- I noticed that `erf` was a term of art and dropped it from the changes, but left the errant corrections in expected results, they're harmless and would be dropped when updating the baseline later).

### Provide values to any of the below to override the defaults.

To use a pull request use a branch name like `pr/N` where `N` is the pull request number.

```
SV_REPO=
SV_BRANCH=
SU_REPO=
SU_BRANCH=
LIBHTP_REPO=
LIBHTP_BRANCH=
```

### CODE REVIEW

File: doc/userguide/reputation/ipreputation/ip-reputation.rst [hunk 1]
```diff
@@ -8,7 +8,7 @@ IP Reputation
...
...to define the feeds to be subscribed to, provide authentication credentials if required, and give a weight to that feed. The weight can be an overall number or a by category weight. This will allow the admin to minimize the influence a feed has on their overall reputation if they distrust a particular category or feed, or trust another implicitly. Feeds can be configured to accept feedback or not and will report so on connect. The admin can override and choose not to give any feedback, but the sensor should report these to the Hub upstream on connect.  The hub will take all of these feeds and aggregate them into an average single score for each IP or IP Block, and then redistribute this data to all local sensors as configured. It should receive connections from sensors. The sensor will have to provide authentication and will provide feedback. The hub should redistribute that feedback from sensors to all other sensors as well as up to any feeds that accept feedback.  The hub should also have an API to allow outside statistical analysis to be done to the database and fed back into the stream. For instance a local site may choose to change the reputation on all Russian IP blocks, etc.
```
Comment (jsoref) [inline doc/userguide/reputation/ipreputation/ip-reputation.rst:11]:
```diff
-fed back in to the stream
+fed back into the stream
```

File: python/bin/suricatasc [hunk 2]
```diff
@@ -9,7 +9,7 @@
 # This program is distributed in the hope that it will be useful,
 # but WITHOUT ANY WARRANTY; without even the implied warranty of
 # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU General Public License for mo re details.
+# GNU General Public License for more details.
```
Comment (jsoref) [inline python/bin/suricatasc:12]: there were a couple of items w/ damaged license blocks...

File: src/detect-content.c [hunk 3]
```diff
@@ -2291,7 +2291,7 @@ static int SigTest41aTestNegatedContent(void)
  */
 static int SigTest42TestNegatedContent(void)
 {                                                                                                                                                        // 01   5    10   15   20  24
-    return SigTestPositiveTestContent("alert tcp any any -> any any (msg:\"HTTP URI cap\"; content:!\"twentythree\"; depth:22; offset:35; sid:1;)",  (uint8_t *)"one four nine fourteen twentythree thirtyfive fourtysix fiftysix");
+    return SigTestPositiveTestContent("alert tcp any any -> any any (msg:\"HTTP URI cap\"; content:!\"twentythree\"; depth:22; offset:35; sid:1;)",  (uint8_t *)"one four nine fourteen twentythree thirtyfive fortysix fiftysix");
```
Comment (jsoref) [inline src/detect-content.c:2294]: I decided against verifying if the code actually cared about the `forty*` tokens, sorry

File: src/detect-csum.c [hunk 4]
```diff
@@ -941,7 +941,7 @@ static int DetectCsumValidArgsTestParse01(void)
     FAIL_IF_NULL(de_ctx);\
     Signature *s = DetectEngineAppendSig(de_ctx, "alert ip any any -> any any ("mystr(kwstr)"-csum:vaid; sid:1;)");\
     FAIL_IF(s);\
-    s = DetectEngineAppendSig(de_ctx, "alert ip any any -> any any ("mystr(kwstr)"-csum:invaalid; sid:2;)");\
+    s = DetectEngineAppendSig(de_ctx, "alert ip any any -> any any ("mystr(kwstr)"-csum:invalid; sid:2;)");\
```
Comment (jsoref) [inline src/detect-csum.c:944]: Puzzled by this one. I didn't take the time to investigate in detail.

File: src/source-dpdk.c [hunk 5]
```diff
@@ -507,10 +507,10 @@ static TmEcode ReceiveDPDKThreadInit(ThreadVars *tv, const void *initdata, void
         // some PMDs requires additional actions only after the device has started
         DevicePostStartPMDSpecificActions(ptv, dev_info.driver_name);
 
-        uint16_t inconsist_numa_cnt = SC_ATOMIC_GET(dpdk_config->inconsitent_numa_cnt);
-        if (inconsist_numa_cnt > 0) {
+        uint16_t inconsistent_numa_cnt = SC_ATOMIC_GET(dpdk_config->inconsitent_numa_cnt);
+        if (inconsistent_numa_cnt > 0) {
```
Comment (jsoref) [inline src/source-dpdk.c:511]: I can't find the `inconsitent_numa_cnt` token in dpdk: https://github.com/search?q=repo%3ADPDK%2Fdpdk%20inconsitent&type=code

(I sent them PRs a while ago...)

But I left it unchanged under the assumption that it's an upstream element.

Having now searched more carefully, I think I was too cautious here.

File: src/util-mpm.h [hunk 6]
```diff
@@ -68,7 +68,7 @@ typedef struct MpmPattern_ {
     uint8_t *original_pat;
     /* case sensitive */
     uint8_t *cs;
-    /* case INsensitive */
+    /* case Insensitive */
```
Comment (jsoref) [inline src/util-mpm.h:71]: I thought about this for a bit and have decided that the nonstandard casing didn't add anything of value...

File: src/detect-csum.c (hunk 4 above)
Comment (victorjulien) [reply src/detect-csum.c:944]: it appears to be a test to validate a misspelled input is leading to a parsing error

**Comment by victorjulien** [comment]
Merged in #8840, thanks @jsoref. I've added proper thank you's to the commit this time, sorry for not doing that in the first batch.

**Pull request closed without merging**